.changeset/clean-pandas-act.md

```diff
@@ -0,0 +1,58 @@
+---
+'@nhost/nhost-js': minor
```

Comment:
Let's respect semver. I think this should be a major update. What do you think @nunopato?

Reply:
yes

Reply:
I agree. The question is whether we can avoid introducing it now.